### PR TITLE
LPS-124963 Reduced API surface - 2

### PR DESCRIPTION
**Pull request by stian-sigvartsen**
We had a conversation about how best to name the new "util" type. For info see https://github.com/liferay-appsec/liferay-portal/pull/264#pullrequestreview-588778176

### CODE REVIEW

File: modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/util/OpenIdConnectSessionHelperImpl.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/util/OpenIdConnectSessionHelperImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by stian-sigvartsen** [comment]
ci:test:relevant

**Comment by stian-sigvartsen** [comment]
ci:abort

**Comment by stian-sigvartsen** [comment]
ci:test:relevant

**Comment by stian-sigvartsen** [comment]
ci:forward

**Pull request closed without merging**